Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -350,7 +350,7 @@ IPCCommandResult ES::ImportContentEnd(Context& context, const IOCtlVRequest& req
 
 ReturnCode ES::ImportTitleDone(Context& context)
 {
-  if (!context.title_import.tmd.IsValid())
+  if (!context.title_import.tmd.IsValid() || context.title_import.content_id != 0xFFFFFFFF)
```

Comment:
Is that value important enough to get a named constant, or perhaps its own function `context.title_import.IsPending()` (or similar)?
I suppose the part that actually sets the value is not in our HLE, so I'm kinda leaning towards the latter.